tools/build_release.py

```diff
@@ -37,13 +37,13 @@ def main():
         * Update the changelog to show the version of the release
         * Change unreleased in TOC to new version
         * Commit to master
-    * `c:\Python36\python.exe tools\build_release.py --build --update --upload --release`
+    * `c:\\Python36\\python.exe tools\\build_release.py --build --update --upload --release`
```

Comment:
Are you sure you need to escape the backslashes if they are inside the backticks?
At least here in GitHub markdown it doesn't appear to be the case.

`c:\\Python36\\python.exe tools\\build_release.py --build --update --upload --release`
c:\\Python36\\python.exe tools\\build_release.py --build --update --upload --release

Reply:
Either that or turn off that check in flake8 altogether. The new version started complaining that `\P` was an undefined escape sequence.

Reply:
Makes sense, I thought it was in a comment bit it's actually inside a multi-line string literal. I think you got it right.